CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.16.1...master
 
+### Added
```

Comment:
This PR is actually also:
 - Adding new methods to `WebJira`
 - Adding new page objects
 - Adding new memory
 - Appending `ViewIssueAction` with comment remembering capability

all of those are in the `api` package, so maybe they should all be mentioned in the changelog?